color_pose red
color_pose green
color_pose blue